piIOComm.c

```diff
@@ -319,81 +319,22 @@ INT32S piIoComm_sendRS485Tel(INT16U i16uCmd_p, INT8U i8uAddress_p,
 	return 0;
 }
 
-INT32S piIoComm_sendTelegram(SIOGeneric * pRequest_p, SIOGeneric * pResponse_p)
+int piIoComm_sendTelegram(SIOGeneric *msg)
```

Comment:
So all the ioctl's which send a user telegram are only interested in sending a request, they don't care for the response. It would be good if you could make that explicit in the commit message as it would relieve the reviewer from having to piece it together from the code changes.

Reply:
Ok